### PR TITLE
[Docs] Release notes 8.0.0

### PR DESCRIPTION
**Pull request by jcalcaben**
## Description

Updates the compatibility table and changelog with information about the 8.0.0 release.

## Related Issue

Closes PWA-927

## Acceptance 

@cherdman 
@ericerway (optional)

### Verification Stakeholders

Any developer

@cherdman 
@ericerway (optional)

### Verification Steps

1. Open CHANGELOG.md
1. Verify release information is accurate

## Checklist

- [ ] I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,266 +1,210 @@
...
+| Implemented initial UX and workflow for a Forgot Password feature for My Account     | **Feature**  | [#2619][] |
+| Implemented initial code for Wishlist                                                | **Feature**  | [#2620][] |
+| Implemented the ability to translate Venia's header / footer                         | **Feature**  | [#2643][] |
+| Added ability to allow PWA Studio to use a different store to localize CMS Pages     | **Feature**  | [#2649][] |
+| Implemented initial code for Address Book                                            | **Feature**  | [#2653][] |
+| Implemented initial code for an Order History table                                  | **Feature**  | [#2660][] |
+| Updated Venia's button style to match new design                                     | **Update**   | [#2496][] |
+| Added product listing to the MiniCart                                                | **Update**   | [#2506][] |
+| Added a header section to the MiniCart                                               | **Update**   | [#2509][] |
+| Added a foort section to the MiniCart                                                | **Update**   | [#2511][] |
```
Comment (revanth0212) [inline CHANGELOG.md:86]:
```suggestion
| Added a footer section to the MiniCart                                                | **Update**   | [#2511][] |
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,266 +1,210 @@
...
+| Added product link for each product in the MiniCart                                  | **Update**   | [#2549][] |
+| Disabled the visibility of the MiniCart on the checkout page                         | **Update**   | [#2554][] |
+| Added product links to the product listings on the cart page                         | **Update**   | [#2557][] |
+| Updated Venia's filter modal styles to match the new design                          | **Update**   | [#2559][] |
+| Added CSS to handle multiple configurable options                                    | **Update**   | [#2577][] |
+| Implemented consistent error state handling                                          | **Update**   | [#2588][] |
+| Added a sign-in section to the new My Account trigger in the header                  | **Update**   | [#2590][] |
+| Enabled GET for GraphQL queries (but not mutations)                                  | **Update**   | [#2602][] |
+| Added React Refresh to improve development server performance                        | **Update**   | [#2609][] |
+| Improve error handling when using the Sign-in dropdown                               | **Update**   | [#2664][] |
```
Comment (revanth0212) [inline CHANGELOG.md:98]:
```suggestion
| Improved error handling when using the Sign-in dropdown                               | **Update**   | [#2664][] |
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,266 +1,210 @@
...
+| Bumped bl from 3.0.0 to 3.0.1                                            | **Update**  | [#2675][]           |
 
 ## Known issues
 
-- A new Mini Cart is in development which removes the checkout flow in favor of the standalone Checkout page.
-  An issue exists in the checkout section of the old Mini Cart where the **Pay with Card** header does not appear on the Credit Card payment form.
-  This issue is low priority since the old Mini Cart will be replaced with the new one in future releases.
+PWA Studio 8.0.0 is not fully compatible with Magento 2.3.6, which can result in the following issues:
+
+- User is unable to Reset password
```
Comment (revanth0212) [inline CHANGELOG.md:203]: Have we decided on whether to include the forgot password feature in 8.0?

File: magento-compatibility.js [hunk 4]
```diff
@@ -4,6 +4,7 @@
 
 // PWA Studio version -> Magento version.
 module.exports = {
+    '8.0.0': '2.4.0 - 2.4.1',
```
Comment (revanth0212) [inline magento-compatibility.js:7]: Should we also include `2.3.6`?

**Review by revanth0212 — DISMISSED** [review]
Minor stuff, nothing critical.

Nice work @jcalcaben 👍

File: magento-compatibility.js (hunk 4 above)
Comment (jcalcaben) [reply magento-compatibility.js:7]: so it should be `2.3.6 - 2.4.1`?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,266 +1,210 @@
...
-Peregrine extensions points are declared in [`peregrine-declare.js`][].
+#### Shopping Bag feature
 
-| Target name | Description                                                                   |
-| ----------- | ----------------------------------------------------------------------------- |
-| `talons`    | Intercept specific Peregrine talons and wrap them with your custom components |
+In addition to the complete full page cart and checkout experience, this release introduces a new Mini-Cart/Shopping Bag feature.
 
-Wrappable talons:
+This feature appears as a floating modal that appears when you click on the shopping bag icon.
```
Comment (cherdman) [inline CHANGELOG.md:41]: Should read: "This feature is a floating modal that appears when you click on the shopping bag icon."

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,266 +1,210 @@
...
+| Deprecated `getUnionAndInterfaceTypes()` function                                            | **Update**  | [#2663][] |
+| Fixed scaffolding bug that did not include a `pwa-studio` section in the `package.json` file | **Bugfix**  | [#2514][] |
+| Fixed Storybook files for scaffolded projects                                                | **Bugfix**  | [#2708][] |
+
+### UPWARD
+
+| Description                                                                      | Change type | PR        |
+| -------------------------------------------------------------------------------- | ----------- | --------- |
+| Implemented feature that allows HTTP for the Magento backend URL                 | **Feature** | [#2423][] |
+| Updated implementation code to improve Webpagetest score                         | **Update**  | [#2548][] |
```
Comment (cherdman) [inline CHANGELOG.md:150]: "WebPageTest" and not "Webpagetest"

**Review by cherdman — CHANGES_REQUESTED** [review]
Awesome job! Just two super minor updates requested.

File: magento-compatibility.js (hunk 4 above)
Comment (revanth0212) [reply magento-compatibility.js:7]: I feel so but we need to get confirmation from either @jimbo or @dpatil-magento.

**Review by cherdman — APPROVED** [review]
Great job. Thanks!

**Review by jimbo — DISMISSED** [review]
Looks good to me. I'll circle back on the version support question.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,266 +1,209 @@
...
-
-### PWA Studio Fundamentals tutorials
-
-Over the course of 7.0.0 development, the PWA Studio doc site has published introductory tutorials for working with the PWA-Studio tools and libraries.
-
-These tutorials provide steps for common tasks associated with storefront development.
-They cover everything from setting up the initial project to providing a checklist for deploying to production.
-
-See [PWA Studio fundamentals][] for a list of these tutorials.
+During 8.0.0 development, the PWA Studio doc site has published referenced documentation for extensibility targets in the different packages.
```
Comment (supernova-at) [inline CHANGELOG.md:63]:
```suggestion
During 8.0.0 development, the PWA Studio doc site has published reference documentation for extensibility targets in the different packages.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,266 +1,209 @@
...
+| Created components for a Newsletter Subscription page                                | **Feature**  | [#2571][] |
+| Implemented initial code for an Order History page                                   | **Feature**  | [#2611][] |
+| Implemented a way to access top/intermediate level categories through the left nav   | **Feature**  | [#2616][] |
+| Implemented initial UX and workflow for a Forgot Password feature for My Account     | **Feature**  | [#2619][] |
+| Implemented initial code for Wishlist                                                | **Feature**  | [#2620][] |
+| Implemented the ability to translate Venia's header / footer                         | **Feature**  | [#2643][] |
+| Added ability to allow PWA Studio to use a different store to localize CMS Pages     | **Feature**  | [#2649][] |
+| Implemented initial code for Address Book                                            | **Feature**  | [#2653][] |
+| Implemented initial code for an Order History table                                  | **Feature**  | [#2660][] |
+| Updated Venia's button style to match new design                                     | **Update**   | [#2496][] |
```
Comment (supernova-at) [inline CHANGELOG.md:83]: Would it be easier if we added a "Change Type" label on our PRs like we have with versions?

Feature, Update, Refactor, Bugfix?

Comment (jcalcaben) [reply CHANGELOG.md:83]: yeah that would help

File: magento-compatibility.js [hunk 9]
```diff
@@ -4,6 +4,7 @@
 
 // PWA Studio version -> Magento version.
 module.exports = {
+    '8.0.0': '2.3.6 - 2.4.1',
```
Comment (supernova-at) [inline magento-compatibility.js:7]: I can see arguments for both sides of this but seeing as how our official support line has been "last two Magento releases", I think we should stick with that here.

It seems like a happy accident that anything more than that is compatible, and in this case specifically the reset password flow doesn't work. I do like that we've listed that as a known issue if we keep the current compatibility value.

**Comment by dpatil-magento** [comment]
@jcalcaben PR's from release branch are missing in change log, pr 2709 2722 2723 2726 2736 2727 2739 2744.

**Comment by dpatil-magento** [comment]
QA Approved,

**Review by cherdman — DISMISSED** [review]
Looks great. Nice job!

File: magento-compatibility.js (hunk 9 above)
Comment (jimbo) [reply magento-compatibility.js:7]: @supernova-at Agree completely. Support for `2.3.6` is coincidental, so I wouldn't list it.

**Review by jimbo — APPROVED** [review]

**Review by dpatil-magento — APPROVED** [review]

**Review by supernova-at — APPROVED** [review]

**Pull request merged**